webapp/src/features/UserItem/UserInformation.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Col, Row } from 'react-bootstrap'
+import Cards from '../../widgets/Cards'
+import classNames from 'classnames'
+
+const UserInformation = () => {
+	const mockData = [
+		{
+			name: 'Total Revenue',
+			value: '$2,050',
+		},
+		{
+			name: 'Monthly Recurring Revenue',
+			value: '$70',
+		},
+		{
+			name: 'Subscription Status',
+			value: 'Active',
+		},
+		{
+			name: 'Add-Ons',
+			value: '5 states',
+		},
+		{
+			name: 'Role',
+			value: 'Owner',
+		},
+		{
+			name: 'Plan',
+			value: 'Flex',
+		},
+		{
+			name: 'Group',
+			value: 'Brandcave',
+		},
+	]
+	return (
+		<Cards className='mb-25' title='User Information'>
+			<Row className='user-info'>
+				{mockData.map((user, index) => (
```

Comment:
index is used for what?

Reply:
it was not being used, so removed from the code.